### PR TITLE
r-dtplyr: add new package

### PR DESCRIPTION
**Pull request by manuelakuhn**

### CODE REVIEW

**Comment by manuelakuhn** [comment]
ping @glennpj

**Comment by manuelakuhn** [comment]
@spackbot run pipeline

**Comment by manuelakuhn** [comment]
@spackbot run pipeline

File: var/spack/repos/builtin/packages/r-dtplyr/package.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RDtplyr(RPackage):
+    """Data Table Back-End for 'dplyr'.
+    Provides a data.table backend for 'dplyr'. The goal of 'dtplyr' is to
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-dtplyr/package.py]: Add a blank line after the first.
```suggestion
    """Data Table Back-End for 'dplyr'.
    
    Provides a data.table backend for 'dplyr'. The goal of 'dtplyr' is to
```

**Review by glennpj — CHANGES_REQUESTED** [review]

**Review by glennpj — DISMISSED** [review]

**Review by glennpj — APPROVED** [review]
Sorry, github's suggested change feature added extra spaces. Looks good.

**Pull request merged**